### PR TITLE
[Docs] Cleanup broker documentation 

### PR DESCRIPTION
**Pull request by kellyfj**
clarify broker/topic ownership, role of AuthN and AuthZ working together, fix readability of bundle docs

### Motivation
The architecture diagrams generally do not show Brokers being aware of each other but they are not entirely stateless and are aware of which brokers own which namespace bundles. So clarify that as well as cleanup some other areas (e.g. AuthN occurs at proxy not just broker)

### Modifications
Updated documentation

### Verifying this change
This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): no
  - The public API: no
  - The schema: no
  - The default values of configurations: no
  - The wire protocol: no
  - The rest endpoints: no
  - The admin cli options:  no
  - Anything that affects deployment: no

### Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by kellyfj** [comment]
CC: @sijie @addisonj perhaps you can confirm if my understanding of brokers is correct. Please feel free to suggest alternate wording. Thanks in advance.

**Review by jiazhai — APPROVED** [review]

File: site2/docs/admin-api-namespaces.md [hunk 1]
```diff
@@ -417,7 +417,7 @@ admin.namespaces().getNamespaceMessageTTL(namespace)
 
 #### split bundle
```
Comment (Jennifer88huang) [inline site2/docs/admin-api-namespaces.md:418]:
```suggestion
#### Split bundle
```

File: site2/docs/admin-api-namespaces.md [hunk 2]
```diff
@@ -417,7 +417,7 @@ admin.namespaces().getNamespaceMessageTTL(namespace)
...
 
-Each namespace bundle can contain multiple topics and each bundle can be served by only one broker. If bundle gets heavy with multiple live topics in it then it creates load on that broker and in order to resolve this issue, admin can split bundle using this command.
+Each namespace bundle can contain multiple topics and each bundle can be served by only one broker. If a bundle has multiple live heavily-used topics in it, then it creates load on that broker and in order to resolve this issue an admin can split the bundle using this command permitting the new bundles to be spread across the cluster.
```
Comment (Jennifer88huang) [inline site2/docs/admin-api-namespaces.md:420]: 1. what does "If a bundle has multiple live heavily-used topics in it" and "create load" mean? maybe we can refine it to be clearer?
2. Could we refine the last sentence as "... an admin splits the bundle using this command,  and spreads the new bundles across the cluster."?

File: site2/docs/concepts-architecture-overview.md [hunk 3]
```diff
@@ -150,3 +150,5 @@ from pulsar import Client
 
 client = Client('pulsar://pulsar-cluster.acme.com:6650')
 ```
+
+Note that in Pulsar, each topic is handled by only one broker. Therefore, for creating, reading, updating and deleting topics, requests are sent to any broker initially. If the broker finds that it cannot handle the request for this topic, it redirects the client to the correct broker.
```
Comment (Jennifer88huang) [inline site2/docs/concepts-architecture-overview.md:154]: > Note
> In Pulsar, each topic is handled by only one broker. Requests for creating, reading, updating and deleting topics are sent to the broker owner initially. If the broker owner cannot handle the request for this topic, it redirects the request to  another/ the correct broker.

File: site2/docs/admin-api-namespaces.md (hunk 1 above)
Comment (kellyfj) [reply site2/docs/admin-api-namespaces.md:418]: FYI this capitalization is not consistent with the remainder of the document - if you agree I'm happy to make the rest of the document consistent with your proposed capitalization scheme. Thanks!

File: site2/docs/admin-api-namespaces.md (hunk 2 above)
Comment (kellyfj) [reply site2/docs/admin-api-namespaces.md]: Made some suggestions based on your feedback. Thanks!

File: site2/docs/concepts-architecture-overview.md (hunk 3 above)
Comment (kellyfj) [reply site2/docs/concepts-architecture-overview.md]: Thanks - I took your feedback and made a few tweaks to it.

File: site2/docs/admin-api-namespaces.md (hunk 1 above)
Comment (Jennifer88huang) [reply site2/docs/admin-api-namespaces.md]: Yes, we use sentence case style for headings in Pulsar docs.
Thank you very much for your contribution.

File: site2/docs/admin-api-namespaces.md [hunk 4]
```diff
@@ -415,9 +415,10 @@ admin.namespaces().getNamespaceMessageTTL(namespace)
 ```
 
 
-#### split bundle
+#### Split bundle
 
-Each namespace bundle can contain multiple topics and each bundle can be served by only one broker. If bundle gets heavy with multiple live topics in it then it creates load on that broker and in order to resolve this issue, admin can split bundle using this command.
+Each namespace bundle can contain multiple topics and each bundle can be served by only one broker. 
+If a single bundle is creating an excessive load on a broker an admin can split the bundle using this command permitting one or more of the new bundles to be unloaded thus spreading the load across the brokers.
```
Comment (Jennifer88huang) [inline site2/docs/admin-api-namespaces.md:421]:
```suggestion
If a single bundle is creating an excessive load on a broker, an admin splits the bundle using this command permitting one or more of the new bundles to be unloaded thus spreading the load across the brokers.
```

File: site2/docs/concepts-architecture-overview.md [hunk 5]
```diff
@@ -150,3 +150,5 @@ from pulsar import Client
 
 client = Client('pulsar://pulsar-cluster.acme.com:6650')
 ```
+
+Note: In Pulsar, each topic is handled by only one broker. Initial requests from a client to read, update or delete a topic are sent to a broker that may not be the topic owner. If the broker cannot handle the request for this topic, it redirects the request to the appropriate broker.
```
Comment (Jennifer88huang) [inline site2/docs/concepts-architecture-overview.md:154]:
```suggestion
> **Note**
> In Pulsar, each topic is handled by only one broker. Initial requests from a client to read, update or delete a topic are sent to a broker that may not be the topic owner. If the broker cannot handle the request for this topic, it redirects the request to the appropriate broker.
```

File: site2/docs/admin-api-namespaces.md (hunk 1 above)
Comment (Jennifer88huang) [reply site2/docs/admin-api-namespaces.md]: I notice that the capitalization is not consistent, if you could refine the rest of them, that's really great.

File: site2/docs/concepts-architecture-overview.md [hunk 6]
```diff
@@ -150,3 +150,6 @@ from pulsar import Client
 
 client = Client('pulsar://pulsar-cluster.acme.com:6650')
 ```
+
+**Note**
+In Pulsar, each topic is handled by only one broker. Initial requests from a client to read, update or delete a topic are sent to a broker that may not be the topic owner. If the broker cannot handle the request for this topic, it redirects the request to the appropriate broker.
```
Comment (Jennifer88huang) [inline site2/docs/concepts-architecture-overview.md:155]:
```suggestion
> **Note**
> In Pulsar, each topic is handled by only one broker. Initial requests from a client to read, update or delete a topic are sent to a broker that may not be the topic owner. If the broker cannot handle the request for this topic, it redirects the request to the appropriate broker.
```

**Review by Jennifer88huang — APPROVED** [review]
@kellyfj thank you very much for your contributions

**Pull request merged**